src/options_widgets.jl

```diff
@@ -19,8 +19,8 @@ e.g. `togglebuttons(OrderedDict("good"=>1, "better"=>2, "amazing"=>9001))`
 function togglebuttons(labels_values::Associative;
                        ob = nothing, value=nothing, multiselect=false, label="")
     defaultval = multiselect ?
-        Vector{valtype(labels_values)}() : medianelement(labels_values)
-    ob, value = init_wsigval(ob, value; default=defaultval)
+        Vector{Int}() : medianelement(1:length(labels_values))
+    ob, value = init_wsigval(ob, value; default=defaultval, typ=Any)
```

Comment:
Is this a problem if you actually do want the ToggleButtons' obs to be of a specific type?